src/Component/Native/BackgroundColor/index.js

```diff
@@ -1,36 +1,73 @@
-/* eslint-disable class-methods-use-this */
-
-export default class NativeBackgroundColorComponent {
-  constructor(color, el) {
-    this.color = color;
-    this.setContainer(el);
-    this.previousColor = '';
-  }
-
-  /*
-   * We must not mess with the position properties of the style on the container
-   * we are given, or we will break the workbench layout functionality!  Setting the
-   * background color is fine, however, as long as we don't use the setAttribute()
-   * approach to this.  Also, we could always create our own container
-   * within the element we are given, and we can do whatever we want with that.
-   */
-
-  setContainer(el) {
-    if (this.el) {
-      this.el.style['background-color'] = this.previousColor;
+import style from 'PVWStyle/ComponentNative/BackgroundColor.mcss';
+import htmlContent from './body.html';
+import CompositeClosureHelper from '../../../Common/Core/CompositeClosureHelper';
+
+function backgroundColorComponent(publicAPI, model) {
+  publicAPI.resize = () => {
+    if (!model.container) {
+      return;
+    }
+
+    model.clientRect = model.container.getBoundingClientRect();
```

Comment:
I would like the method to be there (so that, for instance, it is easier to copy the component as a starting point for new components). Is it OK for the method to exist but do nothing? Or should I find something "demo-like" for it to do on resize to justify its existence?